sdk/src/main/java/com/karhoo/sdk/api/network/client/APITemplate.kt

```diff
@@ -89,6 +91,8 @@ interface APITemplate {
         const val IDENTIFIER_ORG = "organisation_id"
         const val IDENTIFIER_CURRENCY = "currency"
 
+        const val CHOOSE_PROVIDER = "v3/payments/providers"
```

Comment:
Would it be better to call it `GET_PROVIDERS`?

Reply:
I agree. GET_PROVIDERS sounds more fitting than CHOOSE_PROVIDER for this endpoint and what it will be used for.